sdk/metric/aggregator/minmaxsumcount/mmsc_test.go

```diff
@@ -66,8 +68,8 @@ var (
 func TestMain(m *testing.M) {
 	fields := []ottest.FieldOffset{
 		{
-			Name:   "Aggregator.states",
-			Offset: unsafe.Offsetof(Aggregator{}.states),
+			Name:   "Aggregator.current",
```

Comment:
Since this is no longer using atomics, we don't strictly need to check any offsets for alignment.  Shall I just remove?

Reply:
Sure.

Reply:
Instead of this change, you could just change `Aggregator` to `AggregatorSL`.

Reply:
Ah, wait. AggregatorSL will be dropped anyway. Then this whole `TestMain` function could be dropped.